src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -293,6 +294,8 @@ private static boolean equalsItemMeta(@Nonnull ItemMeta itemMeta, @Nonnull ItemM
             return false;
         } else if (itemMeta.hasDisplayName() && sfitemMeta.hasDisplayName() && !itemMeta.getDisplayName().equals(sfitemMeta.getDisplayName())) {
             return false;
+        } else if (itemMeta instanceof PotionMeta && sfitemMeta instanceof PotionMeta) {
+            return ((PotionMeta) itemMeta).getBasePotionData().hashCode() == ((PotionMeta) sfitemMeta).getBasePotionData().hashCode();
```

Comment:
This overrides any lore checks, so if we have two potions with the same potion data but different lore, they will still pass as "equal", even when the `checkLore` parameter is set to true.

Furthermore `.hashCode()` should not really be used for comparisons like this, there is nothing wrong with using `.equals()` here instead.
Note that  `.hashCode()` and `.equals()` always co-exist with each other and are never overridden by themselves only.
Or as Yoda would put it:

> Always two there are, no more, no less.